assets/app/scripts/directives/logViewer.js

```diff
@@ -141,6 +143,10 @@ angular.module('openshiftConsole')
                 return;
               }
 
+              if(!$scope.run) {
+                return;
+              }
```

Comment:
Great Q.  We have [the issue open for improving error messages](https://github.com/openshift/origin/issues/5913), perhaps we fix it there?  I was going to move to that one next.